addons/binding/org.openhab.binding.seneye/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,110 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="seneye"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
+        xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+
+    <thing-type id="monitor">
+        <label>Seneye Monitor</label>
+        <description>Seneye Monitor</description>
+
+        <channels>
+            <channel id="temperature" typeId="seneye-temperature-channel"/>

```

Comment:
You should leave out the "seneye-" prefix and the "-channel" suffix from all your channel types - this is superfluous and it is no problem if the type id is identical to the channel id.

Reply:
ok